engines/babylonjs/index.js

```diff
@@ -0,0 +1,100 @@
+/** 
+ * Parse parameters from url
+ * @returns - parse parameters
+*/
+function parseParameters() {
+    let result = {};
+    let parameters = location.href.split("?")[1];
+    if (parameters) {
+        parameters = parameters.split("&");
+        for (let i = 0; i < parameters.length; i++) {
+            let parameter = parameters[i].split("=");
+            switch (parameter[0]) {
+                case "assetUrl": {
+                    result.assetURL = parameter[1];
+                    break;
+                }
+                case "cameraPosition": {
+                    result.cameraPosition = parseArray(parameter[1]);
+                    break;
+                }
+            }
+        }
+    }
+
+    return result;
+}
+
+/**
+     * Parse a string into an array of numbers.
+     * @param {string} arr 
+     */
+function parseArray(arr) {
+    let result = [];
+    let entries = arr.substring(1, arr.length - 1).split(",");
+    let length = entries.length;
+
+    for (let i = 0; i < length; ++i) {
+        result.push(Number(entries[i]));
+    }
+
+    return result;
+}
+
+/**
+ * Fetches the manifest data from the url
+ * @param {string} manifestFile - url of the manifest file
+ * @param {function(object)} onSuccess - callback triggered on success with the manifest data as the argument.
+ * @param {*} onError - callback triggered on failure.
+ */
+function getManifest(manifestFile, onSuccess, onError) {
```

Comment:
Is this used?

Reply:
Hmm, not anymore since we are passing the model directly.  I'll remove that.